docs/API_DESIGN.md

```diff
@@ -88,9 +91,11 @@ CLASS VerifiableCredential
   PUBLIC DATA id: string
   PUBLIC DATA type: []string
   PUBLIC DATA issuer: Issuer
+  PUBLIC DATA credentialSubject: CredentialSubject
```

Comment:
we don't really have a good solution for casing in our APID, but maybe this just to be consistent

```suggestion
  PUBLIC DATA credential_subject: CredentialSubject
```